frontend/src/views/ChatView.vue

```diff
@@ -118,7 +123,11 @@ const languages = ref(['english', 'italian', 'french', 'spanish', 'german']);
 
 const selectedDictionary = ref<null | number>(null);
 const dictionaries = ref();
```

Comment:
```suggestion
const dictionaries = ref<Components.Schemas.DictionaryDto[]>();
```
se possibile evitiamo l'utilizzo impicito o esplicito di any